apps/bplan/serializers.py

```diff
@@ -1,4 +1,5 @@
-import os
+import datetime
+import posixpath
```

Comment:
i did copy the _generate_image_filename logic from django where posixpath was used. i am not sure why the prefered it over os.path